README.md

```diff
@@ -16,7 +16,7 @@ The options available to the Customer are view menu, sign up and log in, create
 
 There are several classes that inherit the Base class from SQLAlchemy in order to create tables. 
 
-These classes are Food Category, Food Details, Customer Details, Customer Order Selection, Customer Order Status and Delivery Person.
+These classes are Food Category, Food Details, Customer Details, Customer Order Selection, Customer Order Status and Delivery Person. These classes also include a convert_to_dict method so that the tables can be serialization to a JSON format.
```

Comment:
You can add "..serialization to a JSON format so that it can be used in REST APIs"

Reply:
change mande